eventmesh-common/src/main/java/org/apache/eventmesh/common/utils/IPUtils.java

```diff
@@ -57,22 +58,17 @@ public static String getLocalAddress() {
         //priority of networkInterface when generating client ip
         String priority = System.getProperty("networkInterface.priority", "eth0<eth1<bond1");
 
-        ArrayList<String> preferList = new ArrayList<String>();
-        for (String eth : priority.split("<")) {
-            preferList.add(eth);
-        }
+        ArrayList<String> preferList = new ArrayList<>(Arrays.asList(priority.split("<")));
```

Comment:
It is recommended to split it into two lines. The responsibilities of each line will be clearer.
ArrayList<String> preferList = new ArrayList<String>();
preferList.addAll(Arrays.asList(priority.split("<"));
@weihubeats